zero_bin/leader/src/cli.rs

```diff
@@ -22,6 +24,16 @@ pub(crate) struct Cli {
     // mode.
     #[clap(flatten)]
     pub(crate) prover_state_config: CliProverStateConfig,
+
+    // Mode to use for worker for setup (split or unified)
```

Comment:
Not a fan of the naming, I know what we're doing here and I still have to stop and think

Reply:
@muursh any suggestions?